src/feature/Dashboard/index.jsx

```diff
@@ -1,10 +1,123 @@
-import React from 'react'
-import './Dashboard.module.css'
+import React from 'react';
+import styles from './Dashboard.module.css';
+import logo from "../../images/head.jpg";
+import sniper from "../../images/sniper.png";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser, faUserPlus, faUsers, faIdBadge } from '@fortawesome/free-solid-svg-icons';
+
 
 const Dashboard = () => {
   return (
-    <div>This is the dashboard</div>
-  )
-}
+    <div>
+      <nav className={styles.navbar}>
+        <div className={styles.navbar__left}>
+          <div className={styles.navbar__left__logo}>
+            <img src={sniper} alt="Description" />
+          </div>
+          <div className={styles.navbar__text}>
+            <p>AVI-HUB</p>
+          </div>
+        </div>
+        <div className={styles.navbar__right}>
+          <img src={logo} alt="Description" />
+        </div>
+      </nav>
```

Comment:
The navbar should be a component of its own — Split into a separate component.

The same thing applies with the sidebar.